### ISSUE TITLE
LaunchMethod FUNCS unknown

### ISSUE THREAD

**Issue by AymenFJA** [radical-cybertools/radical.pilot#2612]
This is related to #2153.

In general trying example `12_task_fucntion` fails with the error below:

```python
  File "/home/rp/radical.pilot.sandbox/rp.session.cd09e898-cfd5-11ec-a27c-0242ac110002/pilot.0000/rp_install/lib/python3.6/site-packages/radical/pilot/agent/launch_method/base.py", line 142, in create
    raise ValueError('LaunchMethod %s unknown' % name)
ValueError: LaunchMethod FUNCS unknown
1652127847.769 : agent.0              : 6964  : 139814660425536 : ERROR    : skip LM FUNCS
Traceback (most recent call last):
  File "/home/rp/radical.pilot.sandbox/rp.session.cd09e898-cfd5-11ec-a27c-0242ac110002/pilot.0000/rp_install/lib/python3.6/site-packages/radical/pilot/agent/resource_manager/base.py", line 335, in _prepare_launch_methods
    name, lm_cfg, rm_info, self._log, self._prof)
  File "/home/rp/radical.pilot.sandbox/rp.session.cd09e898-cfd5-11ec-a27c-0242ac110002/pilot.0000/rp_install/lib/python3.6/site-packages/radical/pilot/agent/launch_method/base.py", line 142, in create
    raise ValueError('LaunchMethod %s unknown' % name)
ValueError: LaunchMethod FUNCS unknown
```
As of `devel` there is no `FUNCS` `LM` and only an `EXECUTOR`.

**Comment by andre-merzky** [comment]
Proposal: remove FUNCS executor

**Comment by AymenFJA** [comment]
Fixed with #2677
